src/components/text-editor/prosemirror-adapter/prosemirror-adapter.tsx

```diff
@@ -161,6 +177,7 @@ export class ProsemirrorAdapter {
             'open-editor-link-menu',
             this.handleOpenLinkMenu,
         );
+        this.view.dom.removeEventListener('blur', this.handleBlur);
```

Comment:
For good measure, we should check that `this.view` isn't undefined here, since it's technically possible (though unlikely) that the component is disconnected before we call `this.initializeTextEditor()` inside `componentDidLoad()`

Reply:
⚡ bc22455434a572b8930b5a8a7fa7e0bd58222daf

Reply:
I'll squash it immediately.

Reply:
Thanks for catching that by the way! 😊